recipes/r-rsvg/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set version = '1.1' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-rsvg
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: rsvg_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/rsvg_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/rsvg/rsvg_{{ version }}.tar.gz
+  sha256: 8ec457f1d52894619329650c58e2597b168e3312a1129d7863e23abad7d54e9e
+
+build:
+  number: 0
+  skip: true  # [win32]
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - posix                # [win]
+    - {{native}}toolchain  # [win]
+    - gcc                  # [not win]
+    - librsvg              # [not win]
+
+  run:
+    - r-base
+    - libgcc   # [not win]
+    - librsvg  # [not win]
+
+test:
+  commands:
+    - $R -e "library('rsvg')"  # [not win]
+    - "\"%R%\" -e \"library('rsvg')\""  # [win]
+
+about:
+  home: https://github.com/jeroen/rsvg#readme
+  license: MIT
```

Comment:
How do we want to handle `license_file` here?

Reply:
Our current policy has been to make sure that GPL licenses are properly shipped with conda-forge R binaries, but we do not currently do this for the other licenses because they are templates.

The discussion on how to ship licenses for R packages is in https://github.com/conda-forge/staged-recipes/issues/3234